platform/android/export/export_plugin.cpp

```diff
@@ -2383,6 +2392,12 @@ Error EditorExportPlatformAndroid::sign_apk(const Ref<EditorExportPreset> &p_pre
 	args.push_back("pass:" + password);
 	args.push_back("--ks-key-alias");
 	args.push_back(user);
+
+	if(!userPassword.empty()) {
```

Comment:
```suggestion
	if (!userPassword.empty()) {
```

Reply:
already edited.
Issue 1: Since property names are changed / added, error messages below should be changed accordingly. e.g.
Answer: Corresponding processing has been done in the code. When "user_password" is empty, "password" will be used as the default password, which is compatible with the previous processing method of the document author.
Refer to the following code:
```
// Fix export or run program signature issues.
args.push_back("--key-pass");
if (user_password.is_empty()) {
args.push_back("pass:" + password);
} else {
args.push_back("pass:" + user_password);
}

```
Question 2: Why is this called "User Pass" instead of "Key Pass"?
Answer: The documentation author used "user" for "--ks-key-alias", and to correspond with "user_password" for "--key-pass" .
--This text is translated from the Internet.